papers/bradley_dice/55_bradley_dice.rst

```diff
@@ -0,0 +1,553 @@
+
+In our growth with increasing contributors and users, we recognized a need to change our governance structure to make contributing easier and provide a clear organizational structure to the community.
+We based our new model on the Meritocratic Governance Model and our manager roles on Numba Czars.
+We decided on a four category system with maintainers, committers, contributors, and users.
+Code review and PR merge responsibilities are granted to maintainers and committers, who are (self-)nominated and accepted by a vote of the project maintainers.
+Contributors consist of all members of the community who have contributed in some way to the framework, which includes adding or refactoring code as well as filing issues and improving documentation.
+Finally, users refer to all those who use signac in any capacity.
+
+In addition, to avoid overloading our committers and maintainers, we added three rotating manager roles to our governance model that ensure project management goes smoothly: triage, community, and release.
+These managers have specific rotation policies based on time (or release cycles).
+The triage manager role rotates weekly and looks at new issues or pull requests and handles cleanup of outdated issues.
+The community manager role rotates monthly and is in charge of meeting planning and outreach.
+Lastly, the release manager rotates with each release cycle and is the primary decision maker for the timeline and feature scope of package releases.
+This prevents burnout among our senior developers and provides a sense of ownership to a greater number of people, instead of relying on a "benevolent dictator/oligarchy for life" mode of project leadership.
+
+
+Conclusions
+-----------
+
+From the birth of the signac framework to now, signac has grown in usability, performance, and use.
+Since our last proceedings papers, we have added exciting new features, like groups, aggregates, and synced collections and learned how to better manage outreach and governance in a burgeoning scientific open-source project.
+As maintainers and committers, we are looking to continue expanding the framework through user-oriented development and continued outreach to research fields that routinely have projects suited for signac.
+For example, extensions into experimental research labs are currently being sought after with an aim to provide the strong data management and providence signac provides into experimentalist communities.
+
+Getting signac
+--------------
+
+The signac framework is tested for Python 3.6+ and is compatible with Linux, macOS, and Windows.
+The software is available under the BSD-3 Clause license.
+
+To install, execute
+
+.. code-block:: bash
+
+    conda install -c conda-forge signac \
+    signac-flow signac-dashboard
```

Comment:
We haven't mentioned dashboard at all before this installation section. I'm fine if we remove this for the sake of being consistent with what we've discussed above, but perhaps we should mention signac-dashboard briefly in the Overview of New Features.